browser/src/Plugins/Api/Oni.ts

```diff
@@ -66,6 +67,10 @@ export class Oni extends EventEmitter implements Oni.Plugin.Api {
         return editorManager
     }
 
+    public get input(): Oni.InputManager {
+        return inputManager
```

Comment:
Since this is basically the API definition, I was thinking that:
`Oni.input.bind(..)`
is more readable than:
`Oni.inputManager.bind(..)`